parts/k8s/kubernetesagentcustomdata.yml

```diff
@@ -52,6 +52,26 @@ write_files:
   content: !!binary |
     {{WrapAsVariable "systemConf"}}
 
+- path: /etc/apt/apt.conf.d/10periodic
```

Comment:
Should the same files be added to master custom data?

Reply:
Indeed